hadoop-ozone/common/src/main/bin/ozone

```diff
@@ -69,6 +69,12 @@ function ozonecmd_case
   subcmd=$1
   shift
 
+  ozone_default_log4j="${HADOOP_CONF_DIR}/log4j.properties"
+  ozone_shell_log4j="${HADOOP_CONF_DIR}/ozone-shell-log4j.properties"
+  if [ ! -f ${ozone_shell_log4j} ]; then
```

Comment:
shellcheck:13: note: Double quote to prevent globbing and word splitting. [SC2086]